### ISSUE TITLE
Banana Cake Pop schema not showing correct schema on multiple endpoints

### ISSUE THREAD

**Issue by reinux** [ChilliCream/hotchocolate#4082]
**Describe the bug**
When I run multiple HC endpoints on the same host, BCP shows the same schema on both.

The only way to get it to show the right schema is to clear browser cookies.

**Expected behavior**
Show the schema associated with the endpoint on which BCP is running.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 11.3.4

**Comment by reinux** [comment]
Just realized that you need to change the inner URL to get it to use the right endpoint. This isn't updated when you change the browser URL.

Might it make sense for it to reflect the browser URL rather than whatever's being stored?

**Comment by arif-hanif** [comment]
> Just realized that you need to change the inner URL to get it to use the right endpoint. This isn't updated when you change the browser URL.
> 
> Might it make sense for it to reflect the browser URL rather than whatever's being stored?

banana cake pop is an ide that can work with multiple endpoints your own schema a third party schema and the concept is baked around documents which could point to multiple different end points

**Comment by reinux** [comment]
Right, but what I'm trying to say is that it would be nice for it to recognize when it's being hosted on the same URL as the GraphQL server itself.

**Comment by michaelstaib** [comment]
This will be addressed with v12.
